gitk-git/gitk

```diff
@@ -12631,7 +12631,9 @@ set cdup {}
 if {[expr {[exec git rev-parse --is-inside-work-tree] == "true"}]} {
     set cdup [exec git rev-parse --show-cdup]
 }
-set worktree [exec git rev-parse --show-toplevel]
+if {$hasworktree} {
+    set worktree [exec git rev-parse --show-toplevel]
```

Comment:
Shouldn't we move this into the conditional preceding the added code?

Also: where do we set `worktree` for bare repositories?

Reply:
There is a difference between being **inside** a worktree and not _having_ a worktree at all. 
Bare git repos do not have a worktree and hence that is why it fails. And worktree also does not need to be set for bare git repos either, simply because there is not any.